### PR TITLE
Update wunderground_data.py

### PR DESCRIPTION
**Pull request by uroh**
add Foggy and Fog Early

### CODE REVIEW

File: custom_components/wundergroundpws/wunderground_data.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ class WUndergroundData:
         ],
         'fog': [
             "Fog",
+            "Foggy",
+            "Fog Early",
```
Comment (shtrom) [inline custom_components/wundergroundpws/wunderground_data.py:66]: `Fog Early` is not necessary here, as the `Early` gets stripped by using the `condition_modifiers` below, and it ends up looking like `Fog`, which is already coverred.

**Comment by cytech** [comment]
noted for next release.
would be nice if weather company API documented the responses so we aren't chasing these down forever...

**Comment by uroh** [comment]
ok, i updated to the latest version and removed "Fog early" and added "AM Snw Shwrs" in "snowy". I'm not sure of the classification, however I think it means a snowfall in the morning. Cytech, thanks for the new wunderground code.

**Comment by uroh** [comment]
none,
"Unsupported condition string "AM Snw Shwrs". Please update WUndergroundData.condition_map and/or WUndergroundData.condition_modifiers."

**Comment by uroh** [comment]
That's right, I removed the "AM". Thank you

**Pull request closed without merging**